frontend/tests/utils/privateApi.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-ignore
```

Comment:
@alejsdev what do you think of having this?

currently without this running the build test would fail. The other option would be to ignore the test files when running build, which maybe is better?

Reply:
We could ignore the test files while running the build :)